### DISCUSSION TITLE
In the copy section dialog could there be a checkbox for 'copy markers'

### DISCUSSION THREAD

**Discussion by Sy256** [helge17/tuxguitar#84] [General]
When you go to copy a section of measures could it also have an option to 'include markers'? Optional, because sometimes you don't want it. It would be cool also if it could auto-add a number to those pasted markers when it copies (so you copy verse, prechor, chorus, it pastes as verse 2, prechor 2, chorus 2, if a checkbox is selected) but that's not needed. Thanks!

**Comment by Sy256** [comment]
Hello is this possible you think? I'll close if not.

I know you don't like to add too many checkboxes where a more practical solution could exist but for the incrementing markers another approach could be to just put a 'add number' modifier in the marker > list markers. Where you could have a field for entering a number, and a button to press to append the number to whatever marker is selected. But again the most important part is copying the markers the incrementing is optional

**Comment by guiv42** [comment]
Yes I definitely think that adding a checkbox "copy markers" in copy measures dialog is possible: it's already done ;-)
Available in pre-release 2023-09-23
Note that the states of the 2 checkboxes is now memorized (suggestion from another user)
Auto-add a number is significantly more complex, so I don't plan to implement it.

**Reply by Sy256** [reply]
Oops didn't catch that haha thank you! Closed!
